packages/plugin-lighthouse/src/lib/runner/utils.ts

```diff
@@ -38,43 +39,46 @@ export class LighthouseAuditParsingError extends Error {
   }
 }
 
+function formatBaseAuditOutput(lhrAudit: Result): AuditOutput {
+  const { id: slug, score, numericValue, displayValue } = lhrAudit;
+  return {
+    slug,
+    score: score ?? 1, // score can be null
+    value: numericValue ?? score ?? 0, // not every audit has a numericValue
```

Comment:
❗ An edge case: Audits like `"unsized-images"` whose values are not binary but that also don't have a `numericValue` now display an unformatted score in the value column (e.g., 0.5 when the score is 50%). Should it be reverted (a fallback of `0`)?

Reply:
I think this is fine for the raw value. But it could be a nice improvement to also set `displayValue` using our `formatReportScore` function in this case :thinking:

Reply:
Done ✅ 

An audit `displayValue` should now be a formatted score when `displayValue` is missing and `scoreDisplayMode` is not binary.